### PR TITLE
Update insider-risk-management-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Data leaks by priority users are no longer in preview and are available for the general audience. Updated the Document accordingly.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9671

File: microsoft-365/compliance/insider-risk-management-policies.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Each insider risk management policy created from the **Data leaks** template can
 
 See the [Create, test, and tune a DLP policy](create-test-tune-dlp-policy.md) article for step-by-step guidance to configure DLP policies for your organization.
 
-### Data leaks by priority users (preview)
+### Data leaks by priority users
 
 Protecting data and preventing data leaks for users in your organization may depend on their position, level of access to sensitive information, or risk history. Data leaks can include accidental oversharing of highly sensitive information outside your organization or data theft with malicious intent. With an assigned data loss prevention (DLP) policy as a triggering event option, this template starts scoring real-time detections of suspicious activity and result in an increased likelihood of insider risk alerts and alerts with higher severity levels. Priority users are defined in [priority user groups](insider-risk-management-settings.md#priority-user-groups-preview) configured in the insider risk management settings area.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-policies.md]:
```suggestion
Protecting data and preventing data leaks for users in your organization may depend on their position, level of access to sensitive information, or risk history. Data leaks can include accidental oversharing of highly sensitive information outside your organization or data theft with malicious intent. With an assigned data loss prevention (DLP) policy as a triggering event option, this template starts scoring real-time detections of suspicious activity and results in an increased likelihood of insider risk alerts and alerts with higher severity levels. Priority users are defined in [priority user groups](insider-risk-management-settings.md#priority-user-groups-preview) configured in the insider risk management settings area.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, not related to your change, but could you also commit the additional suggested change, please? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@msbemba validation warning. Please review. Thanks

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder

**Comment by msbemba** [comment]
> @msbemba Gentle reminder

This has been fixed.

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #9671

**Pull request closed without merging**